examples/text-classification/README.md

```diff
@@ -155,3 +155,78 @@ python run_glue.py \
   --use_hpu_graphs_for_inference \
   --bf16
 ```
+
+## Llama Guard for GLUE MRPC task
+
+Llama Guard can be used for text classification task. The *transformers* library will change the head of the model for you during fine-tuning or inference. You can use same general command as for BERT, except you need to add `--add_pad_token=True`, because Llama based models don't have `pad_token` in their model and tokenizer configuration files. So, `--add_pad_token=True` will add `pad_token` to tokenizer and model's config as `eos_token` if it's not defined.
+
+### Fine-tuning with DeepSpeed
+
+Llama Guard can be fine-tuned with DeepSpeed, here is how you would do it on the text classification MRPC task using DeepSpeed with 8 HPUs:
+
+```bash
+python ../gaudi_spawn.py /
+    --world_size 8 --use_deepspeed run_glue.py \
+    --model_name_or_path meta-llama/LlamaGuard-7b \
+    --gaudi_config gaudi_config.json  \
+    --task_name mrpc  \
+    --do_train  \
+    --do_eval \
+    --per_device_train_batch_size 32 \
+    --per_device_eval_batch_size 8  \
+    --learning_rate 3e-5  \
+    --num_train_epochs 3 \
+    --max_seq_length 128 \
+    --add_pad_token True \
+    --output_dir /tmp/mrpc_output/ \
+    --use_habana \
+    --use_lazy_mode \
+    --use_hpu_graphs_for_inference \
+    --throughput_warmup_steps 3  \
+    --deepspeed path_to_my_deepspeed_config
+```
+
+You can look at the [documentation](https://huggingface.co/docs/optimum/habana/usage_guides/deepspeed) for more information about how to use DeepSpeed in Optimum Habana.
+Here is a DeepSpeed configuration you can use to train your models on Gaudi:
+```json
+{
+    "steps_per_print": 64,
+    "train_batch_size": "auto",
+    "train_micro_batch_size_per_gpu": "auto",
+    "gradient_accumulation_steps": "auto",
+    "bf16": {
+        "enabled": true
+    },
+    "gradient_clipping": 1.0,
+    "zero_optimization": {
+        "stage": 2,
+        "overlap_comm": false,
+        "reduce_scatter": false,
+        "contiguous_gradients": false
+    }
+}
+```
```

Comment:
can we use the one from https://github.com/huggingface/optimum-habana/blob/main/tests/configs/deepspeed_zero_2.json ? and remove this?
check how other example README with deepspeed

Reply:
can you send me your ci test cmd and log from run?